playbook/app/pb_kits/playbook/pb_typeahead/typeahead.rb

```diff
@@ -45,6 +47,9 @@ def typeahead_with_pills_options
           placeholder: placeholder
         }
 
+        base_options.merge!({getOptionLabel: get_option_label}) if get_option_label.present?
+        base_options.merge!({getOptionValue: get_option_value}) if get_option_value.present?
+
```

Comment:
It's silly to send `nil` for the prop which will convert to `null` which isn't handled by `react-select` lib.